clojure-ts-mode.el

```diff
@@ -1380,7 +1389,6 @@ See `clojure-ts--font-lock-settings' for usage of MARKDOWN-AVAILABLE."
   (let ((use-markdown-inline (and clojure-ts-use-markdown-inline
                                   (treesit-ready-p 'markdown-inline t))))
     (when use-markdown-inline
-      (treesit-parser-create 'markdown-inline)
```

Comment:
Ah, so this was not supposed to be used in this manner. Good to know!

Reply:
Yes, I actually tried to pass `:local t` to the range function but it didn't work because I didn't remove this line as well. It was a good idea to ask for help in Emacs-bugs mailing list.